pools/hooks/index.json

```
@@ -0,0 +1,18 @@
+[
+    {
+        "id": "hooks_exitfee",
+        "name": "Exitfee Hook",
+        "description": "The exit fee hook",
+        "hooks": {
+            "SEPOLIA": ["0x307d96183f133c738af11d1971bf0a5ee15312be"]
+        }
```

Realise using GqlChain would be easiest for the API consumption, but should the metadata repo be more generic and us chain IDs?
```suggestion
        "addresses": {
            "SEPOLIA": ["0x307d96183f133c738af11d1971bf0a5ee15312be"]
        }
```